src/main/java/org/prebid/server/metric/Metrics.java

```diff
@@ -329,6 +329,22 @@ public void updateAdapterRequestErrorMetric(String bidder, MetricName errorMetri
         forAdapter(bidder).request().incCounter(errorMetric);
     }
 
+    public void updateAdapterRequestDisabledBidderMetric(String bidder, Account account) {
+        forAdapter(bidder).request().incCounter(MetricName.disabled_bidder);
+        if (accountMetricsVerbosityResolver.forAccount(account)
+                .isAtLeast(AccountMetricsVerbosityLevel.detailed)) {
+            forAccount(account.getId()).adapter().forAdapter(bidder).request().incCounter(MetricName.disabled_bidder);
+        }
+    }
+
+    public void updateAdapterRequestUnknownBidderMetric(String bidder, Account account) {
+        forAdapter(bidder).request().incCounter(MetricName.unknown_bidder);
+        if (accountMetricsVerbosityResolver.forAccount(account)
+                .isAtLeast(AccountMetricsVerbosityLevel.detailed)) {
+            forAccount(account.getId()).adapter().forAdapter(bidder).request().incCounter(MetricName.unknown_bidder);
+        }
+    }
```

Comment:
I don't like this solution. Someone can spam the server with invalid bidders and load the memory by creating new entries in the map (with many invalid bidders).